fmriprep/interfaces/confounds.py

```diff
@@ -7,6 +7,8 @@
 
     >>> import os
     >>> import pandas as pd
+    >>> import numpy as np
+    >>> import nibabel as nib
```

Comment:
This import did not appear to work in my `GrandMeanScaling` doctest, thus its repetition below, any thoughts on how these imports are interpreted?

Reply:
I guess this one is not necessary if you are doing it again in GMS doctest